client/my-sites/plan-features/header.jsx

```diff
@@ -192,32 +193,22 @@ export class PlanFeaturesHeader extends Component {
 			return <div className={ classes } />;
 		}
 
-		if ( relatedMonthlyPlan ) {
-			const originalPrice = relatedMonthlyPlan.raw_price * 12;
-			return (
-				<span className="plan-features__header-price-group">
-					<PlanPrice
-						currencyCode={ currencyCode }
-						rawPrice={ originalPrice }
-						isInSignup={ isInSignup }
-						original
-					/>
+		if ( discountPrice ) {
+			if ( ! available ) {
+				return (
 					<PlanPrice
-						currencyCode={ currencyCode }
+						currencyCode={ discountPrice }
```

Comment:
I'm not familiar with this area of code and perhaps _that's why_ this line looks a bit odd to me.

Should `currencyCode` be renamed to be more generic if it's being passed something else than "currency codes"?

Reply:
@simison good spot, that's a mistake actually, I just removed it